### PR TITLE
Trev8939/location los

### PR DESCRIPTION
**Pull request by TADraeseke**
Adds sample demonstrating the use of LocationLineOfSight

### CODE REVIEW

File: java/location-line-of-sight/src/main/java/com/esri/arcgisruntime/sample/locationlineofsight/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        // get a screen point from the motion event
+        android.graphics.Point screenPoint = new android.graphics.Point(Math.round(motionEvent.getX()),
+            Math.round(motionEvent.getY()));
+
+        // convert the screen point to a scene point
+        Point scenePoint = mSceneView.screenToBaseSurface(screenPoint);
+
+        if (mObserverLocation == null) {
+          // set the observer location
+          mObserverLocation = new Point(scenePoint.getX(), scenePoint.getY(), scenePoint.getZ() + mZOffset, scenePoint.getSpatialReference());
```
Comment (shellygill) [inline java/location-line-of-sight/src/main/java/com/esri/arcgisruntime/sample/locationlineofsight/MainActivity.java]: You can get an uncaught error here (and similar line below) as scenePoint can be null - e.g. if you tap off the horizon of the earth (e.g. there's no valid location to return).

Comment (TADraeseke) [reply java/location-line-of-sight/src/main/java/com/esri/arcgisruntime/sample/locationlineofsight/MainActivity.java]: Good catch! Thanks @shellygill

**Comment by TADraeseke** [comment]
@shellygill fix added to catch for point selection not on the surface

**Comment by shellygill** [comment]
Thanks @TADraeseke changes look good.

**Pull request merged**